packages/admin-portal/src/sagas/apps/__stubs__/integration-types.ts

```diff
@@ -1,6 +1,6 @@
-import { PagedResultDesktopIntegrationTypeModel_ } from '@/types/desktop-integration-types'
+import { PagedResultDesktopIntegrationTypeModel } from '@/types/desktop-integration-types'
```

Comment:
And again

Reply:
I thought so too but decided against it because of the rewrite